tox.ini

```diff
@@ -0,0 +1,29 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+[tox]
+requires =
+    tox>=4
+envlist = make-prs
+
+[testenv]
+basepython = python3
+setenv =
+  PY_COLORS=1
+passenv =
+  GITHUB_TOKEN
+
+[testenv:make-prs]
+description = Build the Sphinx docs
+deps = pip-tools
+commands_pre =
+    pip install -r requirements.txt
+commands =
+    python make-prs.py {posargs}
+
+[testenv:fmt]
+description = Apply coding style standards to code
+deps =
+    ruff==0.4.5
+commands =
+    ruff check --select I --fix
+    ruff format --preview .
```

Comment:
Should we set a line-length of 99 here (for ruff check and format), either here or in a pyproject.toml, to be consistent with our Ops repo? I think the default of 89 breaks list comprehensions and the like a bit too aggressively.